static/src/stylesheets/inline/story-package-garnett.scss

```diff
@@ -357,22 +357,26 @@ $fc-item-gutter: $gs-gutter / 4;
         }
 
         &.fc-item--full-media-75-tablet {
-            .fc-item__liveblog-blocks {
-                margin-bottom: 5px;
-            }
-
             .fc-item__container.u-faux-block-link--hover .fc-item__liveblog-blocks__inner {
                 background-color: darken($story-package-card-colour, 5%);
             }
 
             .fc-item__liveblog-blocks__inner {
                 background-color: $story-package-card-colour;
+                margin-left: -5px;
+                margin-right: 5px;
                 padding-left: 5px;
+                position: absolute;
+                bottom: 0;
 
                 &:hover {
                     background-color: darken($story-package-card-colour, 5%);
                 }
             }
+
+            .fc-item__standfirst-wrapper {
```

Comment:
Is it possible to hide it only when `show updates` is set?

Reply:
That should be the case already. We're scoped under `.fc-item--live-updates` here. Although that's a good reminder to check! Thanks

Reply:
Whoops sorry, great!